src/index.ts

```diff
@@ -255,6 +260,14 @@ export function extractArgs<TEvent>(isResponseStreamFunction: boolean, ...args:
  * @param tags The tags associated with the metric. Should be of the format "tag:value".
  */
 export function sendDistributionMetricWithDate(name: string, value: number, metricTime: Date, ...tags: string[]) {
+  if (FORBIDDEN_METRIC_NAME_CHARS.some((c) => name.includes(c))) {
+    throw new MalformedMetricName(
```

Comment:
This would cause the customer's lambda function to crash, which I don't think we'd want to do at any point